### ISSUE TITLE
rpmbuild fails on cents 7

### ISSUE THREAD

**Issue by dakl** [dun/munge#46]
Hi, 

I'm trying to install munge on a centos 7 machine and get the following: 

```
...
Requires(post): /bin/sh
Requires(postun): /bin/sh
Requires: libc.so.6()(64bit) libc.so.6(GLIBC_2.14)(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3.4)(64bit) libc.so.6(GLIBC_2.4)(64bit) rtld(GNU_HASH)
Processing files: munge-debuginfo-0.5.11-1.el7.centos.x86_64
Provides: munge-debuginfo = 0.5.11-1.el7.centos munge-debuginfo(x86-64) = 0.5.11-1.el7.centos
Requires(rpmlib): rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1 rpmlib(CompressedFileNames) <= 3.0.4-1
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/vagrant/rpmbuild/BUILDROOT/munge-0.5.11-1.el7.centos.x86_64
error: Installed (but unpackaged) file(s) found:
   /usr/lib/systemd/system/munge.service
   /usr/lib/tmpfiles.d/munge.conf


RPM build errors:
    Installed (but unpackaged) file(s) found:
   /usr/lib/systemd/system/munge.service
   /usr/lib/tmpfiles.d/munge.conf
```

Any ideas there? 

Thanks

**Comment by dun** [comment]
Duplicate of #33. Already fixed on master. 0.5.12 should be out next week.

**Comment by dakl** [comment]
Great, thanks!
